spec/support/deposit_helpers.rb

```diff
@@ -32,7 +32,6 @@ def deposit_object(filenames: [])
         filenames.first => { 'preserve' => true, 'publish' => false, 'shelve' => false },
         filenames.last => { 'preserve' => false, 'publish' => true, 'shelve' => true }
       }
-      filenames = filenames.map { |file| "spec/fixtures/#{file}" }
       file_set_type_strategy = SdrClient::Deposit::ImageFileSetStrategy
```

Comment:
is this removed because it was never used?

Reply:
fwiw, i didn't have a great sense of the specific motivation for the change, so i ran `spec/features/sdr_deposit_spec.rb` before i merged this (passed for me)